google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreImpl.java

```diff
@@ -249,6 +250,9 @@ public void onResponse(BatchGetDocumentsResponse response) {
             DocumentReference documentReference;
             DocumentSnapshot documentSnapshot;
 
+            // TODO(metrics): record first_response_latency here. Need to come up with a way to
```

Comment:
Some methods in this calls should have added monitoring info, like `recursiveDelete`, `getAll`, `runTransaction`. discuss with Ehsan.